packages/docusaurus-types/src/index.d.ts

```diff
@@ -51,6 +51,7 @@ export interface DocusaurusConfig {
       }
   )[];
   titleDelimiter?: string;
+  noIndex?: boolean;
```

Comment:
As there's a default value, ? should not be needed

Reply:
So why timeDelimter have a defaut value?

Reply:
because it's a mistake :) when we introduced the schema validation/normalization, we added default values but may have forgotten to modify some types accordingly